docs/sources/configure/_index.md

```diff
@@ -2037,6 +2037,11 @@ tsdb_shipper:
   [mode: <string> | default = ""]
 
   [ingesterdbretainperiod: <duration>]
+
+  # Experimental. Whether TSDB should cache postings or not. The
+  # index-read-cache will be used as the backend.
+  # CLI flag: -tsdb.enable-cache-postings
+  [enable_cache_postings: <boolean> | default = false]
```

Comment:
Nit: this should be named `enable_postings_cache`, or simply `cache_postings`. In its current form it doesn't really make sense.

Reply:
hah thank you. pushed a commit addressing it.